lib/ExportsStorage.js

```diff
@@ -296,4 +329,31 @@ export default class ExportsStorage {
       );
     });
   }
+
+  search(variableName) {
+    return new Promise((resolve, reject) => {
+      this.db.all(
+        `
+          SELECT name, path, isDefault, packageName
+          FROM exports_fts WHERE name MATCH ?
+          COLLATE NOCASE
+        `,
+        variableName,
+        (err, rows) => {
+          if (err) {
+            reject(err);
+            return;
+          }
+          resolve(
+            rows.map(({ name, path, isDefault, packageName }) => ({
```

Comment:
If you wanted to DRY up things, you could introduce a normalization function (`normalizeRowData`?) and use it here and inside the `get()` method.